README.md

```diff
@@ -47,6 +49,18 @@ Firstly, you'll need to enable some features. Expand 'Configuration' and select
 
 ![Enable Features](/Images/GraphSyncFeature.PNG)
 
+#### Create Content Item
```

Comment:
This copy would be better in the user guide (linked to below), rather than in the developer environment setup.